### DISCUSSION TITLE
안녕하세요!

### DISCUSSION THREAD

**Discussion by ghost** [lhwdev/covid-selftest-macro#7] [기능 제안]
좋은 소스 공유해 주셔서 감사합니다!
혹시 반의 자가진단 현황 여부를 출력하는 API를 만드실 생각은 없으신가요?

**Comment by lhwdev** [comment]
- 교사 계정이 있을 경우 반의 현황을 보여주는 API는 이미 있습니다. 다만 앱에서 그걸 구현하지 않았을 뿐이죠.
PoC나 [`classManager.kt`](https://github.com/lhwdev/covid-selftest-macro/blob/master/api/src/main/kotlin/com/lhwdev/selfTestMacro/api/classManager.kt)를 보시면 알 수 있습니다.
- 한 학생 계정에서 반의 현황을 불러올 수는 없습니다.
- 반의 학생 모두의 계정이 있다면 할 수는 있겠는데.. transkey 패치 이후로 한 사람 로그인하는데 1초쯤 걸려서(다른 최적화를 전부 한다 해도 네트워크 속도 때문에..) 한참 걸릴 거고 반 애들의 정보를 다 알아야 합니다.

**Reply by ghost** [reply]
답변 감사합니다:)
